server.ts

```diff
@@ -19,17 +19,17 @@ class RefAssetManager implements IAssetManager {
         {
           id: 1,
           title: "Tears of Steel",
-          uri: "https://maitv-vod.lab.eyevinn.technology/tearsofsteel_4k.mov/master.m3u8",
+          uri: "http://localhost:80/sintel_1min_stream/index.m3u8",
```

Comment:
Do not refer to local test assets in the reference implementations

Reply:
reverted back to the original